apps/dotcom/client/e2e/fixtures/Database.ts

```diff
@@ -30,6 +30,7 @@ const defaultUser = {
 	flags: '',
 	locale: null,
 	animationSpeed: null,
+	keyboardShortcuts: true,
```

Comment:
The name makes me think we store actual keyboard shortcuts. Maybe `keyboardShortcutsEnabled` or something like that? In some cases we also use the `isSomething` naming (`isDarkMode`).